packages/compartment-mapper/test/bundle.test.js

```diff
@@ -69,6 +69,54 @@ test('bundles work', async t => {
   t.deepEqual(log, expectedLog);
 });
 
+test('using named evaluate bundles work', async t => {
+  const bundle = await makeBundle(read, fixture, {
+    useNamedEvaluate: 'nestedEvaluate',
+  });
+  const log = [];
+  const print = entry => {
+    log.push(entry);
+  };
+  const compartment = new Compartment({
+    globals: {
+      print,
+      nestedEvaluate(source) {
+        return compartment.evaluate(source);
+      },
```

Comment:
Assuming this is the expected use pattern, are there concerns about the endowment being retrieved and possibly exploited by source code? Although the options for addressing that are limited and messy.

Reply:
Wherein the protagonist discovers that there’s no documentation of makeBundle.